nixos/modules/programs/nano.nix

```diff
@@ -12,16 +12,16 @@ in
 
       nanorc = lib.mkOption {
         type = lib.types.lines;
-        default = "";
+        default = ''
+          set nowrap
+          set tabstospaces
+          set tabsize 2
+          include "${pkgs.nano}/share/nano/*.nanorc"
```

Comment:
It's also to make build artefacts render better; package values tend to not look very good when rendered directly, to the point of being useless to the reader (compare the rendered result of `default = pkgs.foo` versus `defaultText = "pkgs.foo"`).